Project.toml

```diff
@@ -18,7 +18,7 @@ Accessors = "0.1.32"
 Aqua = "0.8"
 BenchmarkTools = "1"
 ClimaComms = "0.6.2"
-ClimaCore = "0.14.23"
+ClimaCore = "0.14.27"
```

Comment:
Could we maybe preserve compatibility with previous version of ClimaCore, but throw an error when the HDF5 writer is used for `PointSpaces` for such versions?